protocol_test.go

```diff
@@ -0,0 +1,59 @@
+package tarantool_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	. "github.com/tarantool/go-tarantool"
+)
+
+func TestProtocolInfoCopyPreservesFeatures(t *testing.T) {
+	original := ProtocolInfo{
+		Version:  ProtocolVersion(100),
+		Features: []ProtocolFeature{ProtocolFeature(99), ProtocolFeature(100)},
+	}
+
+	origCopy := original.Copy()
+
+	original.Features[1] = ProtocolFeature(98)
+
+	require.Equal(t,
+		origCopy,
+		ProtocolInfo{
+			Version:  ProtocolVersion(100),
+			Features: []ProtocolFeature{ProtocolFeature(99), ProtocolFeature(100)},
+		})
+}
+
+func TestOptsCopyPreservesRequiredProtocolFeatures(t *testing.T) {
```

Comment:
It will look better in another file: connection_test.go ?

Reply:
Ok. At first I had put it in the separate file, but then decided not to increase the quantity of files in the root folder. (But it seems that it is ok in Go.)

Reply:
Moved